packages/wasm-sdk/src/sdk.rs

```diff
@@ -82,6 +198,51 @@ impl WasmSdkBuilder {
     }
 }
 
+// Store shared trusted contexts
+use once_cell::sync::Lazy;
+use std::sync::Mutex;
+
+static MAINNET_TRUSTED_CONTEXT: Lazy<Mutex<Option<crate::context_provider::WasmTrustedContext>>> =
```

Comment:
I prefer to avoid statics. Just an opinion, but please double-check if this is necessary.

Reply:
I think it's fine, since it will live the entire session.